Dockerfile

```diff
@@ -1,15 +1,20 @@
 FROM library/node:6.3
 
-EXPOSE 3000
+RUN mkdir /src
+
+WORKDIR /src
+
+COPY package.json .
 
-WORKDIR /usr/src/app
+RUN npm install
 
 COPY . .
 
-# npm install needs to run after the COPY because of postinstall deps.
-RUN npm install --unsafe-perm --allow-root \
-	&& npm cache clean \
-	&& node_modules/bower/bin/bower --allow-root cache clean
+RUN node_modules/bower/bin/bower --allow-root install \
```

Comment:
I think it should work — bower would create a dir and then the rest will be merged when copying?
If not we can first `mkdir -p` it and ensure the source directory is `.dockerignore`d just in case (if building locally for example).

Also NIT: the path can be `node_modules/.bin/bower` —